### DISCUSSION TITLE
Nova 4 - renaming Main dashboard works only with `label()` instead of `name()` method.

### DISCUSSION THREAD

**Discussion by bastihilger** [laravel/nova-issues#3847] [Q&A]
- Laravel Version: 9.2
- Nova Version: 4.0.0
- PHP Version: 8.0
- Database Driver & Version: MySQL 8.0
- Operating System and Version: MacOS Monterey
- Browser type and version: all
- Reproduction Repository: https://github.com/###/###

### Description:

Hi there, while I can use the `public function name()` in other dashboards than the Main dashboard, on the main dashboard I have to overwrite the method `label()`. In the docs it only mentions the `name()` method - I guess that should be changed?

**Comment by schonhoff** [comment]
I struggled with it yesterday, too. It  is only wrong in the Main Dashboard. On other dashboards you need to use the `name()` method.

The method used in `laravel\nova\src\Dashboards\Main.php` should be `name()` instead of `label()`, because `label()` is calling the `name()` method of the abstract Dashboard class.

abstract Dashboard class:
```php
    /**
     * Get the displayable name of the dashboard.
     *
     * @return string
     */
    public function label()
    {
        return $this->name();
    }
```
